user-console/src/main/java/org/pentaho/mantle/client/admin/EmailAdminPanel.java

```diff
@@ -41,12 +42,27 @@ public class EmailAdminPanel extends SimplePanel {
   protected CheckBox useSSLCheckBox;
   protected TextBox fromAddressTextBox;
   protected TextBox fromNameTextBox;
-  protected CheckBox authenticationCheckBox;
   protected TextBox userNameTextBox;
   protected PasswordTextBox passwordTextBox;
   protected Button testButton;
   protected Button saveButton;
   protected VerticalPanel authenticationPanel;
+  protected TextBox tokenUrlTextBox;
+  protected VerticalPanel oauthPanel;
+  protected VerticalPanel passwordPanel;
+  protected ListBox authenticationTypeLB;
+  protected TextBox clientIdTextBox;
+  protected PasswordTextBox clientSecretTextBox;
+  protected TextBox scopeTextBox;
+  protected ListBox grantTypeListBox;
+  protected TextBox refreshTokenTextBox;
+  protected VerticalPanel refreshTokenPanel;
+  protected TextBox authorizationCodeTextBox;
+  protected VerticalPanel authCodePanel;
+  protected TextBox redirectUriTextBox;
+  protected VerticalPanel smtpPanel;
+
+  private static final String PANEL_WIDTH = "220px";
```

Comment:
good that you created a const. As of now 220 is a magic number, just give a reasoning why its 220 in a javadoc comment.